src/components/betting-chips/BettingChip.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { observer } from "mobx-react";
+
+import styles from "./BettingChip.module.scss";
+import { useMainStore } from "../../hooks/useMainStore";
+
+export type BettingChipProps = {
+  value: string;
+  color: string;
+};
+
+export const BettingChip = observer((props: BettingChipProps) => {
+  const { value, color } = props;
+  const { game } = useMainStore();
+
+  const addToBet = () => {
+    game.setCurrentBet(+value);
+  };
+
+  return (
+    <button
+      className={styles.chip}
+      style={{ borderColor: `${color}` }}
```

Comment:
You can completely drop **color** prop and change color via CSS by reading value of chip. Like this https://www.w3schools.com/css/css_attribute_selectors.asp